src/XrdHttp/XrdHttpReq.cc

```diff
@@ -858,11 +805,10 @@ void XrdHttpReq::parseResource(char *res) {
   resourceplusopaque = resource;
   // Whatever comes after is opaque data to be parsed
   if (strlen(p) > 1) {
-    buf = unquote(p + 1);
+    const char * buf = decode_str(p + 1).c_str();
```

Comment:
There's a [warning](https://my.cdash.org/viewBuildError.php?type=1&buildid=2913086) here that needs to be fixed. It's a dangling pointer to a temporary, which could cause crashes at runtime.